ui/src/common/components/CitationSummaryGraph/CitationSummaryGraph.jsx

```diff
@@ -157,21 +158,19 @@ class CitationSummaryGraph extends Component {
           <LoadingOrChildren loading={loading}>
             <ErrorAlertOrChildren error={error}>
               <Row type="flex" align="middle">
-                <Col span={1}>
-                  <div className="axis-label y-axis">Papers</div>
-                </Col>
-                <Col span={23}>
+                <Col span={24}>
                   <FlexibleWidthXYPlot
-                    height={300}
+                    height={GRAPH_HEIGHT}
+                    margin={GRAPH_MARGIN}
                     xType="ordinal"
                     yDomain={[0, yDomainMax * 1.3]}
                   >
-                    <HorizontalGridLines />
                     <XAxis
                       className="x-axis"
+                      title="Citations"
```

Comment:
this might collide with the bars of the last range

Reply:
let's see on `QA`, don't wanna spent time to mock the data so that bars appear and I can fix it.